### PR TITLE
IDEA-250689 Check new patch/shelf format

### PR DESCRIPTION
**Pull request by nika-qa**

### CODE REVIEW

**Comment by nika-qa** [comment]
rgergergerg

**Comment by nika-qa** [comment]
regergergerg

**Comment by nika-qa** [comment]
```java
public
```

File: src/pack/TestClass.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package pack;
+
```
Comment (veronikatimkina) [inline src/pack/TestClass.java:2]: egergergergerg

**Pull request merged**